src/transformStateWithClones.js

```diff
@@ -8,6 +8,37 @@
  */
 function transformStateWithClones(state, actions) {
   // write code here
```

Comment:
Remove all comments from the code.

Reply:
![image](https://user-images.githubusercontent.com/126285904/236618071-15926071-e481-4156-b666-82a99bac87e6.png)